data/colors/vars/component_dark.ts

```diff
@@ -209,6 +209,9 @@ export default {
   treeViewItem: {
     chevron: {
       hoverBg: alpha(get('scale.gray.2'), 0.12),
+    },
+    folder: {
+      fill: '#4579CA'
```

Comment:
Is there a scale value this can reference?

Reply:
We currently use `scale-blue-3` for directory icons in dotcom: https://cs.github.com/github/github/blob/e7f27d4cf8902dc3c9e6d8d908f066db54ab4020/app/assets/stylesheets/bundles/global/hx_color-modes.scss?q=org%3Agithub+--color-icon-directory#L34

```suggestion
      fill: get('scale.blue.3')
```